src/actions/sources.js

```diff
@@ -207,6 +207,10 @@ export function selectSource(id: string, options: SelectSourceOptions = {}) {
       [PROMISE]: (async () => {
         await dispatch(loadSourceText(source.toJS()));
         await dispatch(setOutOfScopeLocations());
+        const src = getSource(getState(), id).toJS();
+        if (src.text && isMinified(src.id, src.text)) {
```

Comment:
We should also be checking if this is a generated file or not before auto pretty printing. That's probably the issue with the tests

Reply:
adding  `&&  !isOriginalId(src.id)` should do help

Reply:
thanks @bomsy I'm on it

Reply:
done